pom.xml

```diff
@@ -4,15 +4,16 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>error-code-model-java</artifactId>
-    <version>0.1.0</version>
+    <version>1.0.0
+    </version>
```

Comment:
Is it intended?

Reply:
The line break not. 1.0.0 yes. It's ready so time to switch to 1.0.0.